### ISSUE TITLE
Stuck on "Loading...", no query "Scrum report"

### ISSUE THREAD

**Issue by johndpalm** [mikaelkrief/VSTS-Extension-ScrumReport#7]
I just installed your extension from the Marketplace.  I'm running TFS 2015 Update 2 on-premises.  After installing to my DefaultCollection, when I navigate to Work/Scrub Report a "Loading..." message appears and nothing else happens.  I've left it running for quite some time now.  I looked to see if the "Scrum report" showed any results but there isn't a query by that name.

**Comment by mikaelkrief** [comment]
HI @john wich version of the extension you downloaded ?

**Comment by johndpalm** [comment]
The one currently on the Marketplace: 1.8

**Comment by mikaelkrief** [comment]
since 10 minutes there is a new version 1.9 compatible with TFS 2015 Update 2.
Redownload it and update it on your server

Keep me informed if its OK

**Comment by johndpalm** [comment]
Uninstalled 1.8, installed 1.9.  The "loading" message is gone but I still get an error:

> Error the query is not found :
> Create the query 'Scrum Report' on the root on Shared Queries folder

I confirm that there isn't a query by that name.

**Comment by mikaelkrief** [comment]
have you a custom template ?
could you use a developper tools (like in chrome) and send me a screenshoot of the console .

**Comment by johndpalm** [comment]
Yes, there were some modifications made to the template.  I tried it on our "sandbox" collection which is not modified.  The Scrum Report query was created and the report works as well.  It looks like it's my problem, not yours.  If I manually create the "Scrum Report" query in the collection with the problem, the report should work, correct?

**Comment by mikaelkrief** [comment]
Hi John,

Yes, create the query named "Scrum Report" into the root on Shared folder and it's will be work.
Here the query that be created by the extension for exemple

![scrum report query](https://cloud.githubusercontent.com/assets/2725302/14200570/0eea2c1e-f7ec-11e5-9d01-8c0cb59422e8.PNG)

**Comment by ElieAouad** [comment]
Hi -
I am having the same issue and tried to uninstall/Reinstall the latest package. When I click under Scrum Report, I see the Loading message and nothing happens. Please let me know what more information you require.

Thanks.

**Comment by ElieAouad** [comment]
Hi - More info!
I changed the query from State Change date to Changed Date, and had to Modify the State to "To Do,In Progress,Review" and it works. When I add "Done" to the state, the problem comes back.

**Comment by sandeepk1977** [comment]
Using in VSTS, but this report never works. Always stuck in Loading...
